_posts/platform/app/2000-01-01-filesystem.md

```diff
@@ -32,6 +32,7 @@ will be lost when the app is restarted.
 If you need to store files in a persistent manner, the good practice is to use
 an external storage service solution like:
 
+* [Outscale OSU](https://wiki.outscale.net/pages/viewpage.action?pageId=43060977) (S3 compatible API)
```

Comment:
Isn't this link (https://wiki.outscale.net/display/EN/About+OSU) better than the one you added?

Reply:
suggestion please

Reply:
```suggestion
* [Outscale OSU](https://wiki.outscale.net/display/EN/About+OSU) (S3 compatible API)
```